pipeline-runner/DESCRIPTION

```diff
@@ -18,7 +18,8 @@ Suggests:
     testthat (>= 3.0.0)
 Imports:
     data.table,
-    covr
+    covr,
+    batchelor
```

Comment:
are you using this?

Reply:
yes. This is an addtional package needed for fastmnn